Lib/ufoLib/glifLib.py

```diff
@@ -242,10 +242,14 @@ def getGLIF(self, glyphName):
 			needRead = True
 		if needRead:
 			fileName = self.contents[glyphName]
-			if not os.path.exists(path):
-				raise KeyError(glyphName)
-			with open(path, "rb") as f:
-				text = f.read()
+			try:
+				with open(path, "rb") as f:
+					text = f.read()
+			except IOError as e:
+				# FileNotFoundError
+				if e.errno == 2:
+					raise KeyError(glyphName)
+				raise e
```

Comment:
you shouldn't need to re-raise the exception, just a `raise` will simple release the last exception on the stack